### DISCUSSION TITLE
[Help] Rail component with mobile screen

### DISCUSSION THREAD

**Discussion by landall** [fomantic/Fomantic-UI#1465] [Elements (button, image, label, loader, ...)] [answered]
# Help Wanted

How can I fit mobile screen with right/left rail？
Can I display it in front of the main container or show it as a drawer?

**Comment by lubber-de** [comment, marked as answer]
If you want a drawer like feature you probably look at the [`sidebar` module](https://fomantic-ui.com/modules/sidebar.html) instead.
To showing a rail infront of the main container, try [`internal rail` ](https://fomantic-ui.com/elements/rail.html#internal)
As stated in the docs already, `rail` has a quite wide default width, thus isn't supposed to be used on mobile screens (at least not mobile phones in portrait mode)
I suggest to use the [`grid` collection ](https://fomantic-ui.com/collections/grid.html) instead depending of your usecase, which also already has proper breakpoint support

**Reply by landall** [reply]
Thanks for your answer. It helps me a lot!
